src/main/java/seedu/eduke8/question/QuizQuestionsManager.java

```diff
@@ -5,13 +5,16 @@
 
 import java.util.ArrayList;
 import java.util.Random;
-// import java.util.logging.Logger;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 public class QuizQuestionsManager {
     public static final String INVALID_QUIZ_QUESTION_NUMBER = "Number of quiz questions must be more than 1";
     public static final String INSUFFICIENT_TOPIC_QUESTIONS_FOR_QUIZ =
             "There is not enough questions in the topic for the quiz!";
 
+    private static Logger logger = Logger.getLogger("Main");
```

Comment:
Will try to abstract this into another class, thanks for implementing with this for now